docs: hipaa excluded features (#3441)
neondatabase/website: content/docs/security/hipaa.md

Audit events may not be logged if database endpoints experience exceptionally heavy load, as we prioritize database availability over capturing log events.

## Non-HIPAA-compliant Neon features

> Neon’s **Data API** feature is hosted outside of Neon’s HIPAA-compliant infrastructure and should not be used with HIPAA-enabled Neon projects.

Author: Thank you. Updated as suggested.

Audit events may not be logged if database endpoints experience exceptionally heavy load, as we prioritize database availability over capturing log events.

## Non-HIPAA-compliant features

> The following Neon features are not part of our HIPAA-compliant infrastructure and should not be used with HIPAA-enabled projects:

Author: ```suggestion
The following features are not yet HIPAA compliant, so cannot be used in projects containing HIPAA data.
```

- **Neon Auth** – Integrates an Auth provider that is not currently HIPAA-compliant.
- **Neon's Data API feature** – Hosted outside of Neon’s HIPAA-compliant environment.

> 

Author: ```suggestion
If you would like an update on when these features will be HIPAA-compliant, please contact [hipaa@neon.tech](mailto:hipaa@neon.tech).

```

Audit events may not be logged if database endpoints experience exceptionally heavy load, as we prioritize database availability over capturing log events.

## Non-HIPAA-compliant features

The following features are not yet HIPAA compliant, so cannot be used in projects containing HIPAA data.

- **Neon Auth** – Integrates an Auth provider that is not currently HIPAA-compliant.
- **Neon's Data API feature** – Hosted outside of Neon’s HIPAA-compliant environment.

> If you would like an update on when these features will be HIPAA-compliant, please contact [hipaa@neon.tech](mailto:hipaa@neon.tech).

Author: ```suggestion

If you would like an update on when these features will be HIPAA-compliant, please contact [hipaa@neon.tech](mailto:hipaa@neon.tech).
```
